sw/device/tests/autogen/meson.build

```diff
@@ -53,3 +53,54 @@ sw_tests += {
     'library': plic_all_irqs_test_lib,
   }
 }
+
+alert_test_lib = declare_dependency(
```

Comment:
thanks! I mean to do that. Somehow totally forgot.. Getting too excited when the auto-gen works haha.

Reply:
Eh.. My edit seems to affect the plic_all_irqs_test:
```
ERROR: /home/vsts/work/1/s/sw/device/tests/BUILD:608:19: Label '//sw/device/tests:autogen/plic_all_irqs_test.c' is invalid because 'sw/device/tests/autogen' is a subpackage; perhaps you meant to put the colon here: '//sw/device/tests/autogen:plic_all_irqs_test.c'?
```
I wonder if I should move the `plic_all_irqs_test` bazel build to autogen/BUILD as well?

Reply:
You could. Once Meson is gone it's very likely we'll be deleting the `autogen` directory and having Bazel generate the test files, which is why I put them in the outer directory. (In general, assume that checked-in autogenerated code will be deleted and generated at build time by Meson).